io-engine/src/lvs/lvs_lvol.rs

```diff
@@ -114,6 +115,16 @@ impl Display for PropName {
     }
 }
 
+/// Resize context to be passed as callback args pointer to spdk.
```

Comment:
We have `cb_arg` and `done_cb` generic helper functions in spdk-rs/src/ffihelper.rs to wrap/unwrap a sender alongside with any payload into an SPDK-style callback arg. We use them extensively. Probably they would work here as well.